### PR TITLE
Fixes #4757 - Localize Text tool confirm and cancel buttons

### LINKED ISSUE #4757: [Annotations][Text Tool] The "Confirm" and "Cancel" buttons are not localized
[Affected versions]:
- Firefox Release 61.0.2 and up

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Have a German Firefox build installed or extracted.

[Steps to reproduce]:
1. Open latest Firefox browser with the profile from prerequisites and navigate to any website.
2. Perform a selection on the page and save it.
3. Click the "Edit this image" button.
4. Click the "Add Text" button and observe the "Confirm" and "Cancel" buttons.  

[Expected result]:
- The "Confirm" and "Cancel" buttons are translated in German.

[Actual result]:
- The "Confirm" and "Cancel" buttons are displayed in English.

[Notes]:
- However, the tooltip for the "Cancel" and "Confirm" buttons is correctly localized.
- Here is a screenshot with the issue:
confirm and cancel

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: locales/en-US/server.ftl [hunk 1]
```diff
@@ -227,10 +227,10 @@ textSizeSmall = Small
 textSizeMedium = Medium
 textSizeLarge = Large
 # Confirm and Cancel button title shown when using text tool
-annotationTextConfirmButton =
-    .title = Confirm
-annotationTextCancelButton =
-    .title = Cancel
+annotationTextConfirmButton = Confirm
```
Comment (flodolo) [inline locales/en-US/server.ftl]: Unfortunately both these strings need a new ID.

Comment (punamdahiya) [reply locales/en-US/server.ftl]: Ah good to know, @flodolo will the updated ID work

**Comment by punamdahiya** [comment]
@johngruen including to give feedback (confirm vs confirm text) on confirm and cancel button alt text. We are using 'confirm selection' for button in crop tool but can keep it 'confirm' for text tool  if 'confirm text' is confusing.

**Comment by punamdahiya** [comment]
@chenba one line change to update ids in text tool for localization. Please review. Thanks!

**Comment by punamdahiya** [comment]
Updated to keep string values as 'Confirm' and 'Cancel'. Thanks for feedback!

**Pull request merged**